components/authentication/authentication.js

```diff
@@ -99,11 +133,36 @@ function postForgotPassword(req,res){
     })
 }
 
+const validationGuard = (req,res, next) =>{
+    const errors = validationResult(req)
+    if(!errors.isEmpty())
+        return res.status(400).json({err : errors.array()})
+    next()
+}

```

Comment:
I think we can move this to its own file,thoughts?

Reply:
Done